src/modules/keyboardmanager/common/Shortcut.h

```diff
@@ -169,10 +172,10 @@ class Shortcut
     int GetCommonModifiersCount(const Shortcut& input) const;
 
     // Function to check if the two shortcuts are equal or cover the same set of keys. Return value depends on type of overlap
-    static KeyboardManagerHelper::ErrorType DoKeysOverlap(const Shortcut& first, const Shortcut& second);
+    static Helpers::ErrorType DoKeysOverlap(const Shortcut& first, const Shortcut& second);

```

Comment:
This method should not be named `DoKeysOverlap` since there is already `EditorHelpers::DoKeysOverlap` that does check keys, this one instead checks shortcuts, so it should be called `DoShortcutsOverlap`.

Reply:
If I'm not mistaken, this method is not using anything from this class, if that is the case, it should be a static method and not belong to this class.

Reply:
It uses private members of Shorctut.